docs/maintain-guides-how-to-vote-councillor.md

```diff
@@ -0,0 +1,46 @@
+---
+id: maintain-guides-how-to-vote-councillor
+title: Voting for Councillors
+sidebar_label: Voting for Councillors
+---
+
+The council is an elected body of on-chain accounts that are intended to represent the passive stakeholders of Polkadot and/or Kusama. The council has two major tasks in governance: proposing referenda and vetoing dangerous or malicious referenda. For more information on the council, see the [governance page](learn-governance#council). This guide will walk you through voting for councillors in the elections.
+
+## Voting for Councillors
+
+Voting for councillors require you to lock your DOTs / KSMs for the duration of your vote. Like the validator elections,
+you can approve up to 16 different councillors and your vote will be equalized among the chosen group. Unlike validator
+elections, there is no unbonding period for your reserved tokens. Once you remove your vote, your tokens will be liquid
+again.
+
+> Warning: It is your responsibility not to put your entire balance into the reserved value when
+> you make a vote for councillors. It's best to keep _at least_ a few KSM to pay for transaction
+> fees.
+
+Go to the [Polkadot Apps Dashboard](https://polkadot.js.org/apps) and click on the "Council" tab. On the right side
+of the window there are two blue buttons, click on the one that says "Vote."
+
+![](assets/council/vote.png)
+
+Since the council uses approval voting, when you vote you signal which of the validators you approve of and your voted
+tokens will be equalized among the winning candidates. Select up to 16 council candidates by moving the slider to
+"Aye" for each one that you want to be elected. When you've made the proper configuration submit your transaction.
+
+![](assets/council/vote_for_yourself.png)
+
+You should see your vote appear in the interface immediately after your transaction is included.
+
+## Removing your Vote
+
+In order to get your reserved tokens back, you will need to remove your vote. Only remove your vote when you're done
+participating in elections and you no longer want your reserved tokens to count for the councillors that you approve.
+
+Go to the "Extrinsics" tab on [Polkadot Apps Dashboard](https://polkadot.js.org/apps).
+
+Choose the account your want to remove the vote of and select the "electionsPhragmen -> removeVoter()" options and
+submit the transaction.
+
+![](assets/council/remove_vote.png)
+
+When the transaction is included in a block you should have your reserved tokens made liquid again and your vote will
+no longer be counting for any councillors in the elections.
```

Comment:
Should we clarify here that the councilors are not harmed by this and that once voted in they're good for a year?

Reply:
Sure we can clarify that it starts at the next term but I don't want to put a definite time period on it yet until we have a better idea what it should be. Right now it's 100 blocks and in the next upgrade it will be 2 hours.

Reply:
I see. Ok, see [here](https://github.com/w3f/polkadot-wiki/issues/136) for an idea how to tackle this stuff that is easily outdated.